# IDEA:
The average travel time of alternative bus route is not significant different from origin X8 bus route.

# Null Hypothesis
The average travek time of alternative bus route is equal to the X8 bus.

# Alternative Hypothesis
The 
The average travel time of alternative bus route  is not equal to the  X8 bus route.

# Significance Level
I define the  significance level as α = 0.05

# Interpretation of the Z value
if Z < -1.96 or Z > 1.96, then reject Null Hypothesis.
Otherwise, we can't reject Null Hypothesis.


In [1]:
from __future__  import print_function
import pandas as pd
import numpy as np
import os

In [10]:
!wget https://raw.githubusercontent.com/fedhere/PUI2018_fb55/master/Lab4_fb55/times.txt

--2018-10-02 21:57:19--  https://raw.githubusercontent.com/fedhere/PUI2018_fb55/master/Lab4_fb55/times.txt
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 151.101.200.133
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|151.101.200.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1179 (1.2K) [text/plain]
Saving to: ‘times.txt.3’

times.txt.3         100%[===================>]   1.15K  --.-KB/s    in 0s      

2018-10-02 21:57:19 (237 MB/s) - ‘times.txt.3’ saved [1179/1179]



In [12]:
data = pd.read_table('times.txt.3', header = None)
data.head()

0
0  31.622239
1  32.821376
2  30.229101
3  31.413766
4  39.010550

In [13]:
u_0 = 36
time_av = float(data.mean())
x = len(data)
sigma = float(data.std())
z = (time_av - u_0)/(sigma/(np.sqrt(x)))
print(z)
if abs(z) > 1.96:
    print('reject the Null Hypothesis')
else:
    print('can not reject the Null Hypothesis')

-2.14905157561
reject the Null Hypothesis


The Z value is -2.14905157561 which is smaller than -1.96. In this case, we need to reject the Null Hypothesis.